composer.json

```diff
@@ -26,8 +27,9 @@
   },
   "require": {
     "omnipay/common": "^3",
+    "ext-json": "*",
     "ext-bcmath": "*",
-    "ext-json": "*"
+    "php": "^7.2"
```

Comment:
nie jestem pewny czy to załatwi sprawę php ^8.0
główna biblioteka ma "php": "^7.2|^8.0"
więc w naszej nie musimy ustawiać wersji PHP

Reply:
dołożyć to 8.0